src/mut/morphology.cpp

```diff
@@ -411,38 +411,32 @@ void Morphology::write(const std::string& filename)
         LBTHROW(UnknownFileType(_err.ERROR_WRONG_EXTENSION(filename)));
 }
 
-bool Morphology::_compare(const Morphology& other, bool verbose) const
```

Comment:
`compare` looked just as good to me (I would also make it a free method, but that's probably a matter of taste).

Reply:
good point, I'll change that